docs/content/docs/tutorials/ec2-example.md

```diff
@@ -0,0 +1,450 @@
+ 
+
+### Prerequisites
+ 
+This guide assumes that you have:
+ 
+- Created an EKS cluster with Kubernetes version 1.16 or higher.
+- Have access to Amazon ECR
+- AWS IAM permissions to create roles and attach policies to roles.
+- Installed the following tools on the client machine used to access your Kubernetes cluster:
+  - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html) - A command line tool for interacting with AWS services.
+  - [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) - A command line tool for working with Kubernetes clusters.
+  - [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) - A command line tool for working with EKS clusters.
+  - [Helm 3.8+](https://helm.sh/docs/intro/install/) - A tool for installing and managing Kubernetes applications.
+  - [Docker](https://docs.docker.com/engine/install/) - A tool to build, share, and run containers.
+ 
+ 
+### Install ACK EC2-Controller
+ 
+Deploy the EC2-Controller using the Helm chart, [ec2-chart](https://gallery.ecr.aws/aws-controllers-k8s/ec2-chart). Note, this example creates resources in the `us-west-2` region, but you can use any other region supported in AWS.
+ 
+* Log into the Helm registry that stores the ACK charts:
+ 
+```bash
+aws ecr-public get-login-password --region us-west-2 | helm registry login --username AWS --password-stdin public.ecr.aws
+```
+
+* Install Helm chart:
+ 
+```bash
+SERVICE=ec2
+RELEASE_VERSION=$(curl -sL "https://api.github.com/repos/aws-controllers-k8s/${SERVICE}-controller/releases/latest" | grep '"tag_name":' | cut -d'"' -f4)
+helm install --create-namespace -n ack-system oci://public.ecr.aws/aws-controllers-k8s/ec2-chart "--version=${RELEASE_VERSION}" --generate-name --set=aws.region=us-west-2
+```
+ 
+For a full list of available values in the Helm chart, refer to [values.yaml](https://github.com/aws-controllers-k8s/ec2-controller/blob/main/helm/values.yaml).
+ 
+ 
+### Configure IAM permissions
+ 
+The controller requires permissions to invoke EC2 APIs. Once the service controller is deployed [configure the IAM permissions](https://aws-controllers-k8s.github.io/community/docs/user-docs/irsa/) using the value `SERVICE=ec2` throughout. The recommended IAM Policy for EC2-Controller can be found in [recommended-policy-arn](https://github.com/aws-controllers-k8s/ec2-controller/blob/main/config/iam/recommended-policy-arn).
+ 
+ 
+### [Optional] Create a VPC and Subnet
+ 
+This section is optional and will NOT be using a single manifest file to deploy the VPC and Subnet. The purpose of this section is to demonstrate a simple use case to shed light on some of the functionality before jumping into a more complex deployment.
+ 
+* Create the **VPC** using the provided YAML and `kubectl apply`:
+```
+cat <<EOF > vpc.yaml
```

Comment:
I did not set up IAM permissions and deployed the VPC. The State column is empty. I can do a describe and get the error but there has to be a state

`dsaurya@3c063003223c ~ % k get vpc
NAME                ID    STATE
vpc-tutorial-test`

Reply:
I would also call out the error one would get if IAM permissions are not set up.